docs/the-new-architecture/pillars-turbomodule.md

```diff
@@ -442,6 +479,21 @@ RCT_REMAP_METHOD(add, addA:(NSInteger)a
 @end
 ```
 
+The final result should look like this:
```

Comment:
I would have only one of this tree view at the end probably?

Reply:
make sense, I'll do it soon

Reply:
much better now @cortinico 🎉